src/hdf5_back.cc

```diff
@@ -1541,6 +1541,33 @@ void Hdf5Back::CreateTable(Datum* d) {
   schemas_[d->title()] = dbtypes;
 }
 
+std::map<std::string, DbTypes> Hdf5Back::ColumnTypes(std::string table) {
```

Comment:
does this function belong in this PR? perhaps it should be rebased on the current develop head?